tests/utils/print_vec_mat.hpp

```diff
@@ -207,5 +213,174 @@ void print_matrix( const grb::Matrix< T, B > & mat, size_t _limit = 0, const cha
 	std::cout << "==============" << std::endl << std::endl;
 }
 
+
+/**
+ * @brief Prints up to \p _limit rows and columns of matrix \p mat with optional heading \p head.
```

Comment:
no need for `@brief` (the first line is always interpreted as the brief description)-- also elsewhere